app/src/main/java/com/example/poste/activities/RegisterActivity.java

```diff
@@ -95,8 +95,6 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
         });
         backButton.setOnClickListener(v -> {
```

Comment:
This method can simply call `onBackPressed()`, which is the behavior if a user presses the phone's back button. It automatically calls finish(), and, since we might later modify that behavior, would be most consistent to ensure we call it here

Reply:
Done.